js/id/renderer/mapillary_image_layer.js

```diff
@@ -1,5 +1,10 @@
-iD.MapillaryLayer = function (context) {
-    var enable = false,
+iD.MapillaryImageLayer = function (context) {
+    var roundedProjection = iD.svg.RoundProjection(context.projection),
+        urlSearch = 'https://a.mapillary.com/v2/search/s/geojson',
+        urlImage = 'http://mapillary.com/map/im/',
+        urlThumb = 'https://d1cuyjsrcm0gby.cloudfront.net/',
+        clientId = 'NzNRM2otQkR2SHJzaXJmNmdQWVQ0dzoxNjQ3MDY4ZTUxY2QzNGI2',
```

Comment:
please change to 
    clientId='NzNRM2otQkR2SHJzaXJmNmdQWVQ0dzo1ZWYyMmYwNjdmNDdlNmVi'